runtime/altair/src/lib.rs

```diff
@@ -1716,6 +1722,26 @@ impl pallet_swaps::Config for Runtime {
 	type SwapId = pallet_foreign_investments::SwapId<Runtime>;
 }
 
+parameter_types! {
+	pub const TokenMuxPalletId: PalletId = cfg_types::ids::TOKEN_MUX_PALLET_ID;
+}
+
+impl pallet_token_mux::Config for Runtime {
+	type AssetRegistry = OrmlAssetRegistry;
+	type BalanceIn = Balance;
+	type BalanceOut = Balance;
+	type BalanceRatio = Ratio;
+	type CurrencyId = CurrencyId;
+	type LocalAssetId = LocalAssetId;
+	type OrderBook = OrderBook;
+	type OrderId = OrderId;
+	type PalletId = TokenMuxPalletId;
+	type RuntimeEvent = RuntimeEvent;
+	type Tokens = OrmlTokens;
+	// TODO(william): Change to weights once they exist
```

Comment:
Can be in another PR, but we should not forget this!

Reply:
Yes, will fix this as part of the RC1